liqd_product/assets/scss/components/_list_item.scss

```diff
@@ -6,6 +6,7 @@
     background-color: $body-bg;
     margin-bottom: $spacer;
     page-break-inside: avoid;
+    word-break: break-word;
```

Comment:
Don't we normally use overflow wrap as it always breaks word?

Reply:
Haha, me saying, that I always find it when need be wasn't true then. Will check!